src/main/java/uk/gov/hmcts/reform/sendletter/exception/UnauthenticatedException.java

```diff
@@ -11,4 +11,13 @@
     public UnauthenticatedException(String message) {
```

Comment:
Do we need this constructor or can we remove it ?

Reply:
This is used for missing header